cifar/android/CifarETTrainingDemo/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this?

Reply:
oh maybe it's optional, since string.xml doesn't have it?